src/useCountUp.ts

```diff
@@ -0,0 +1,94 @@
+import { CallbackProps, CommonProps, UpdateFn } from './types';
+import { useEffect, useRef } from 'react';
+import { createCountUpInstance } from './common';
+import { CountUp as CountUpJs } from 'countup.js';
+
+export interface useCountUpProps extends CommonProps, CallbackProps {
+  startOnMount?: boolean;
+  ref: string | React.RefObject<any>;
+}
+
+const defaults: Partial<useCountUpProps> = {
```

Comment:
I think that we can move `defaults` to `common` file:
```js
export const defaults = {
  decimal: '.',
  decimals: 0,
  delay: null,
  prefix: '',
  separator: '',
  start: 0,
  startOnMount: true,
  suffix: '',
  useEasing: true,
}
```

and use it in both `CountUp` and `useCountUp`

Reply:
This is meant to be a ts-only PR, and I did make this change in my other PR where the component actually uses the hook.